openmp/libomptarget/include/Shared/PluginAPI.h

```diff
@@ -57,8 +57,18 @@ int32_t __tgt_rtl_init_device(int32_t ID);
 // return NULL. Otherwise, return a pointer to the built address table.
 // Individual entries in the table may also be NULL, when the corresponding
 // offload region is not supported on the target device.
-__tgt_target_table *__tgt_rtl_load_binary(int32_t ID,
-                                          __tgt_device_image *Image);
+int32_t __tgt_rtl_load_binary(int32_t ID, __tgt_device_image *Image,
+                              __tgt_device_binary *Binary);
```

Comment:
This is ABI breaking.

Reply:
I thought we determined at some point that the plugin ABI was free to be broken because it's not exported anywhere, and we should only ever load the corresponding plugin from the associated install.

Reply:
Oh, right.